iosevka.rb

```diff
@@ -3,16 +3,57 @@
 class Iosevka < Formula
   desc "Monospace font family for programming built from code"
   homepage "https://be5invis.github.io/Iosevka"
-  url "https://github.com/be5invis/Iosevka/archive/v2.2.0.tar.gz"
-  sha256 "af573349647976d714facbbc8ff13ee14bd050571919aedb5c90681a6f831966"
+  url "https://github.com/be5invis/Iosevka/archive/v3.7.1.tar.gz"
+  sha256 "ed7aee70af326ef22458f8f7603c599fd2f5164ae8062fbd6fb7d56864d98aca"
   head "https://github.com/be5invis/Iosevka.git"
 
+  option "without-widths-normal", "skip building the normal width variant"
+  option "without-widths-extended", "skip building the extended width variant"
+
   option "without-sans", "skip building the default sans variant"
   option "with-slab", "additionally build the serif variant"
   option "with-woff", "additionally build the webfonts (woff)"
   option "with-woff2", "additionally build the webfonts (woff2)"
   option "with-unhinted", "additionally build unhinted TTF"
 
+  option "with-no-ligation", "disable ligations"
```

Comment:
should it be `without-ligation` and `without-cv-ss`?

I guess it makes sense to have it this way to keep iosevka's options intact, but from brew's perspective without- would be more "idiomatic" ^^

I'm fine with either just wanted to bring it up...